WalletWasabi/Helpers/StartupChecker.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.Helpers
+{
+	public static class StartupChecker
+	{
+		public static async Task<bool> ValidateAsync()
```

Comment:
I don't think that you do any sort of "validation".

Reply:
Yeah, but the word **Check** is used in too many places already. If someone got a better name for it, I'll gladly change it.

Reply:
> Yeah, but the word **Check** is used in too many places already.

Do you want to have consistently incorrect naming or ...? I don't get it. 

IMO it's much better to use some longer name like `GetCurrentValuesAsync` and hope that somebody will come and come up with a better shorted name than to use a word that means semantically something totally different.

Reply:
I will go with `GetCurrentValue` for now.